cetmix_tower_server/tests/test_reference_mixin.py

```diff
@@ -92,3 +92,16 @@ def test_search_by_reference(self):
         # Search using malformed (case sensitive)
         search_result = self.ServerTemplate.get_by_reference("not_much_template")
         self.assertEqual(len(search_result), 0, "Result should be empty")
+
+    def test_search_tag_by_reference(self):
```

Comment:
This function is inherited from mixin and is already tested there. We are not modifying it, so no need to add tests for it too.

Reply:
Thanks, I'll remove it